modules/search/search-service/README.md

```diff
@@ -0,0 +1,58 @@
+
+## Parameters
+
+| Name                              | Type     | Required | Description                                                            |
+| :-------------------------------- | :------: | :------: | :--------------------------------------------------------------------- |
+| `prefix`                          | `string` | No       | Prefix of Azure Cognitive Search Resource Name                         |
+| `name`                            | `string` | No       | Name of the Azure Cognitive Search Resource                            |
+| `location`                        | `string` | No       | Location of the Azure Cognitive Search Resource                        |
+| `tags`                            | `object` | No       | Tags of the Azure Cognitive Search Resource                            |
+| `sku`                             | `object` | No       | SKU of the Azure Cognitive Search Resource                             |
+| `authOptions`                     | `object` | No       | AuthOptions of the Azure Cognitive Search Resource                     |
+| `disableLocalAuth`                | `bool`   | No       | DisableLocalAuth of the Azure Cognitive Search Resource                |
+| `disabledDataExfiltrationOptions` | `array`  | No       | DisabledDataExfiltrationOptions of the Azure Cognitive Search Resource |
+| `encryptionWithCmk`               | `object` | No       | EncryptionWithCmk of the Azure Cognitive Search Resource               |
+| `hostingMode`                     | `string` | No       | HostingMode of the Azure Cognitive Search Resource                     |
+| `networkRuleSet`                  | `object` | No       | NetworkRuleSet of the Azure Cognitive Search Resource                  |
+| `partitionCount`                  | `int`    | No       | PartitionCount of the Azure Cognitive Search Resource                  |
+| `publicNetworkAccess`             | `string` | No       | PublicNetworkAccess of the Azure Cognitive Search Resource             |
+| `replicaCount`                    | `int`    | No       | ReplicaCount of the Azure Cognitive Search Resource                    |
+| `semanticSearch`                  | `string` | No       | SemanticSearch of the Azure Cognitive Search Resource                  |
+
+## Outputs
+
+| Name       | Type     | Description                                     |
+| :--------- | :------: | :---------------------------------------------- |
+| `id`       | `string` | ID of the Azure Cognitive Search Resource       |
+| `endpoint` | `string` | Endpoint of the Azure Cognitive Search Resource |
+| `name`     | `string` | Name of the Azure Cognitive Search Resource     |
+
+## Examples
+
+### Example 1
+
+Deploys a Azure Cognitive Search Service
+
+```
+param location string = 'eastus'
+
+module searchService 'br/public:search/search-service:1.0.0' = {
```

Comment:
The version number will always start with `MAJOR.MINOR.1`, which is a bit confusing, but is the default setting for https://github.com/dotnet/Nerdbank.GitVersioning. We will change it to make PATCH version number start with `0` in the future.
```suggestion
module searchService 'br/public:search/search-service:1.0.1' = {
```

Reply:
Fixed